meter.go

```diff
@@ -42,11 +42,14 @@ type Meter struct {
 
 // Mark updates the total.
 func (m *Meter) Mark(count uint64) {
-	if count > 0 && atomic.AddUint64(&m.accumulator, count) == count {
-		// The accumulator is 0 so we probably need to register. We may
-		// already _be_ registered however, if we are, the registration
-		// loop will notice that `m.registered` is set and ignore us.
-		globalSweeper.Register(m)
+	if count > 0 {
+		n := atomic.AddUint64(&m.accumulator, count)
```

Comment:
any reason for this change?

Reply:
No, leftover from debugging. Will revert.